src/managing-workflow/platform-monitoring.md

```diff
@@ -52,18 +52,30 @@ Grafana will preload several dashboards to help operators get started with monit
 These dashboards are meant as starting points and don't include every item that might be desirable to monitor in a
 production installation.
 
-Deis Workflow monitoring does not currently write data to the host filesystem or to long-term storage. If the Grafana
-instance fails, modified dashboards are lost. Until there is a solution to persist this, export dashboards and store
-them separately in version control.
+Deis Workflow monitoring by default does not write data to the host filesystem or to long-term storage. If the Grafana instance fails, modified dashboards are lost. 
+
+### On Cluster Persistence
+
+If you wish to have persistence for Grafana you can set `enabled` to `true` in the `values.yaml` file before running `helm install`.
+
+* ` grafana:`
+* `   # Configure the following ONLY if you want persistence for on-cluster grafana`
+* `   # GCP PDs and EBS volumes are supported only`
+* `   persistence:`
+* `     enabled: true # Set to true to enable persistence`
+* `     size: 5Gi # PVC size`
+
+You have to set (if you do not have it already) `standard` StorageClass as per [PVC Dynamic Provisioning](#pvc-dynamic-provisioning), as it does not get set by default in Kubernetes v1.4.x and v1.5.x.
+
 
 ### Off Cluster Grafana
 
-It is recommended that users provide their own installation for Grafana if possible. The current deployment of Grafana within Workflow is not durable across pod restarts which means custom dashboards that are created after startup will not be restored when the pod comes back up. If you wish to provide your own Grafana instance you can set `grafana_location` in the `values.yaml` file before running `helm install`.
+If you wish to provide your own Grafana instance you can set `grafana_location` in the `values.yaml` file before running `helm install`.
 
 ## InfluxDB
 
 InfluxDB writes data to the host disk, however, if the InfluxDB pod dies and comes back on
-another host the data will not be recovered. We intend to fix this in a future release. The InfluxDB Admin UI is also
+another host the data will not be recovered, you need to enable on-cluster persistence for data to persist. The InfluxDB Admin UI is also
```

Comment:
probably better to switch the comma for a period here, as they are two complete sentences.

> if the InfluxDB pod dies and comes back on another host, the data will not be recovered. You need to enable on-cluster persistence for data to persist.